Fix for WSL / Remote Development

bilgorajskim opened the pull request:
Great extension! However it didn't work for me in WSL2. These changes should fix the issue. Tested on Windows/WSL2, but the extension should now work with any kind of remote workspace.

89netraM commented:
Hello, thanks for contributing!  
You've done a lot of different things. What specifically is it that will fix the problem? I'd appreciate it if you removed all but the necessary change.

bilgorajskim commented:
1. Added `extensionKind` field to package.json, which tells VS Code that this extension should be installed locally and not in remote environment. In WSL2's case, the local environment is Windows, and remote environment is a Linux VM. The extension needs to run on Windows to be able to play audio.
2. Changed how `Uri`s are compared. Previously `Uri` objects were compared by reference. For some reason, it doesn't work in remote workspaces. I could see that there was an `Uri` object inside `e.uris` that matched `window.activeTextEditor.document.uri`'s properties, but comparing them by reference didn't work. I assumed that is because these are not the same objects, but different objects with identical properties. So I changed the code so that it compares `Uri` objects by `.path` property instead.
3. Removed `errorsForActiveFile` function because I wasn't needed after my changes. If not removed, it would need to be modified, changing `errorUris.includes(editor.document.uri)` to `errorUris.find(uri => uri.path === editor.document.uri.path)`. Please let me know if you prefer to keep it.

89netraM reviewed (approved):
Okay, great work 👍  
Thanks!
